File: ECommerceAPI/ECommerceAPI/Models/Image.cs
using System.ComponentModel.DataAnnotations;

namespace ECommerceAPI.Models
{
    public class Image
    {
        public int ImageId { get; set; }
        public string Path { get; set; }
        [Required(ErrorMessage = "name_required")]
        public string Name { get; set; }
        public bool Active { get; set; }
        public int ProductId { get; set; }
    }
}

File: ECommerceFront/src/app/models/image.ts
export class Image {
    image: File;
    name: string;
    active: boolean;
    path: any;
    constructor(image: File = null, name: string = null, path: string = null) {
        this.image = image;
        this.name = name;
        this.path = path;
    }
}
